yksmp/src/lib.rs

```diff
@@ -14,11 +14,15 @@ use std::error;
 struct Function {
     addr: u64,
     record_count: u64,
+    stack_size: u64,
 }
 
-struct Record {
-    offset: u32,
-    live_vars: Vec<LiveVar>,
+pub struct Record {
```

Comment:
Similarly these are all very x86 centric, probably unnecessarily so?

Reply:
`size` we later populate with the functions stack size which is also u64 in LLVM's stackmap format.